src/patternfly/components/Popover/popover.scss

```diff
@@ -95,25 +84,21 @@
   box-shadow: var(--pf-c-popover__arrow--BoxShadow);
 }
 
-.pf-c-popover__close {
-  position: absolute;
-  top: var(--pf-c-popover__close--Top);
-  right: var(--pf-c-popover__close--Right);
-}
-
-// Header
-.pf-c-popover__header {
-  margin-bottom: var(--pf-c-popover__header--MarginBottom);
+.pf-c-button {
+  top: var(--pf-c-popover--c-button--Top);
```

Comment:
Is this `top` necessary? I think it's in the modal, too.

Reply:
I added it because without this negative shift the button looks visually out of line with the title because of the padding applied to `.pf-c-button`. But I assume we're going to be addressing this depending on what we do with the close button.